kinto/core/events.py

```diff
@@ -29,6 +29,7 @@ def __init__(self, payload, request):
         self.request = request
 
     def __repr__(self):
+        # Not sure what proper conversion to fstring is
```

Comment:
This is probably better to leave it with format()

Reply:
We could do this, I think it should work

`    def __repr__(self):
        return f"<{self.__class__.__name__} {payload['action']} {payload['uri']}>"`